src/commands/command.js

```diff
@@ -27,10 +27,34 @@ export default class DiscordCommand {
       this.discordClient = discordClient;
   }
 
-  get hasMentions() {
+  get member() {
```

Comment:
I figured that we don't really care if the bot is used unless it is within a server with members

Reply:
I'm using `member` so that I can have access to `displayHexColor` and `displayName`, which `user` doesn't have.
The hex color and display name are used to make the embeds a little nicer